docs/website/versioned_docs/version-3.0.0/command-reference/delete.md

```diff
@@ -1,40 +1,39 @@
 ---
-title: odo delete
+title: odo delete component
 sidebar_position: 4
 ---
 
-`odo delete` command is useful for deleting resources that are managed by odo.
+`odo delete component` command is useful for deleting resources that are managed by odo. It deletes the component and it's related innerloop and outerloop resources from the cluster.
 
-## Deleting a component
-
-To delete a _devfile_ component, you can execute `odo delete`.
+There are 2 ways to deleting a component:
+1. [Delete with access to Devfile](/#Delete with access to Devfile)
+2. [Delete without access to Devfile](/#Delete without access to Devfile)
 
+## Delete with access to Devfile
 ```shell
-odo delete
+odo delete component
 ```
+odo analyzes the devfile for the devfile component and the outerloop resources.
+If the component has been deployed on the cluster, then odo will list all the resources and prompt the user to confirm the deletion.
+Otherwise, odo will exit with a message stating that it could not find the resources on the cluster.
 
-If the component is pushed to the cluster, running the above command will delete the component from the cluster, and it's dependant storage, url, secrets, and other resources.
-If it is not pushed, the command would exit with an error stating that it could not find the resources on the cluster.
-
-Use `-f` or `--force` flag to avoid the confirmation questions. 
-
-## Un-deploying Devfile Kubernetes components
+_Note:_ odo does not delete the Devfile, and config files.
```

Comment:
Please use the [note admonition](https://docusaurus.io/docs/markdown-features/admonitions). You are italicizing it, someone else might make it bold, a third one could make it both bold & italicized, so on and so forth.

Reply:
> odo does not delete the Devfile, and config files.

Also, which config files are talking about here? The ones under `.odo` directory? And, IMO, it's grammatically incorrect. It could be:

> odo does not delete the Devfile, code or the config files.